soh/src/code/z_map_exp.c

```diff
@@ -664,6 +664,7 @@ void Minimap_Draw(GlobalContext* globalCtx) {
                     if (CHECK_DUNGEON_ITEM(DUNGEON_MAP, mapIndex)) {
                         gDPSetPrimColor(OVERLAY_DISP++, 0, 0, 100, 255, 255, interfaceCtx->minimapAlpha);
 
+                        gSPInvalidateTexCache(OVERLAY_DISP++, interfaceCtx->mapSegment);
```

Comment:
Could this be placed in a more conservative location? Such as when `interfaceCtx->mapSegment` changes, so perhaps at the end of `Map_InitData`.

Generally, texture cache invalidates should happen as infrequently as possible for maximum performance.

Reply:
@Baoulettes Any luck finding a more conservative location?

Reply:
Right now I am not sure if that finding better place.
I have no idea if opening disp and doing an InvalidateTexCache  is better but if it is I would have done it in the end of Map_initData like Rozelette suggested work fine
I would have done it this way :

`            OPEN_DISPS(globalCtx->state.gfxCtx, "../z_map_exp.c", 626);
            gSPInvalidateTexCache(OVERLAY_DISP++, interfaceCtx->mapSegment);
            CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_map_exp.c", 782);`
But I am not sure of the impact of it